src/foam/nanos/approval/ApprovalRequest.js

```diff
@@ -742,6 +742,8 @@
                     createPredicate: foam.mlang.predicate.False.create(),
                     deletePredicate: foam.mlang.predicate.False.create()
                   }),
+                  mementoHead: null,
+                  backLabel: 'Back'
```

Comment:
Can we support translation on this?

Reply:
It will be used in an ActionView, which will translate it, as long as there is a message in the LocaleDAO.